exercises/J-functions/exercise2.js

```diff
@@ -1,5 +1,6 @@
 function triple(number) {
   // complete function here
+  return number *=3
```

Comment:
you don't need extra assignment operator.  "return (number*3);" is enough for best practice. In addition, You can use formatter and leave space between operator  to make code more readable. The others are great, nice coding.